src/pages/index.js

```diff
@@ -1,31 +1,15 @@
 import React, { useEffect } from 'react';
 
-// import React, { Suspense } from "react";
-// import { Redirect, Router } from "@reach/router";
 import { navigate } from 'gatsby';
 import { useTranslation } from 'react-i18next';
 
-import { i18next } from '../components';
-
-function Home({ location }) {
+function Home() {
   const { i18n } = useTranslation();
 
-  // TODO: which method is preferred?
-
-  // method one
   useEffect(() => {
     navigate(`/${i18n.language}/`);
   }, []);
   return <></>;
-
-  // method two throws an error (there's not a page or function yet at /en) – takes time to load
-  // return (
-  // <Router>
-  {
-    /* <Redirect from="/" to={`/${i18n.language}/`} noThrow/> */
-  }
-  // </Router>
-  // );
```

Comment:
Not sure if the changes here will affect the changes I made in my #48.

Reply:
I think we'll have to resolve a merge conflict